.github/workflows/checks-unix.yml

```
@@ -20,7 +20,7 @@ permissions: read-all
 jobs:
   fuzz:
     name: Fuzz
-    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@main
+    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@c-shell
```

This change should be reverted prior to merging:

```suggestion
    uses: ericcornelissen/shescape/.github/workflows/reusable-fuzz.yml@main
```